common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -22,7 +22,15 @@
 from ...pages.lms.course_info import CourseInfoPage
 from ...pages.lms.learner_profile import LearnerProfilePage
 from ...pages.lms.tab_nav import TabNavPage
-from ...pages.lms.teams import TeamsPage, MyTeamsPage, BrowseTopicsPage, BrowseTeamsPage, CreateOrEditTeamPage, TeamPage
+from ...pages.lms.teams import (
+    TeamsPage,
+    MyTeamsPage,
+    BrowseTopicsPage,
+    BrowseTeamsPage,
+    TeamManagementPage,
+    TeamPage
+)
+from ...pages.studio.utils import confirm_prompt
```

Comment:
It would be good to move this to a more general util file, now that we have the same prompt being used by both LMS and CMS.